extensions/2.0/Vendor/EXT_feature_metadata/README.md

```diff
@@ -358,20 +356,15 @@ _Class and feature texture_
           }
         }
       },
-      "featureTextures": {
-        "heatLossTexture": {
-          "class": "heatSample",
-          "properties": {
-            "heatLoss": {
-              "texture": {
-                "index": 0,
-                "texCoord": 0
-              },
-              "channels": "r"
-            }
-          }
+      "featureTextures": [{
+        "class": "heatSample",
+        "index": 0,
+        "texCoord": 0,
+        "properties": {
+          "heatLoss": "r",
+          "insulation": "g"
```

Comment:
Based on today's discussion, I think we want these to use channel indices rather than channel names?

```jsonc
"properties": {
  "heatLoss": 0,
  "insulation": 1,
}
```

... wouldn't be trying to allow GeoTIFF directly here, but this leaves the door open for a future `EXT_texture_geotiff` extension if needed.

Reply:
Should these be strings still? In case we have vec3 properties.

```
"properties": {
  "windVector": "012",
}
```

@ptrgags @sanjeetsuhag do we actually have examples of feature textures that store vector properties? Otherwise maybe we should keep it simple like the snippet above.

Reply:
Hm if we're trying to leave the door open for a potential `EXT_texture_geotiff` someday, then `rgb` --> `012` would not be enough, a GeoTIFF could have >10 bands. [0,1,2] would be another option, and (IMO) doesn't add significant complexity.

Reply:
The array form looks good to me

Reply:
Updated!